src/autocomplete/typeahead.js

```diff
@@ -31,6 +31,7 @@ function Typeahead(o) {
   this.openOnFocus = !!o.openOnFocus;
   this.minLength = _.isNumber(o.minLength) ? o.minLength : 1;
   this.autoWidth = (o.autoWidth === undefined) ? true : !!o.autoWidth;
+  this.clearOnSelected = !!o.clearOnSelected;
```

Comment:
What about using `Boolean(o.clearOnSelected)` instead of `!!o.clearOnSelected`?

Reply:
this is in the same style as the other options, technically both are the same :)

Reply:
They are the same, indeed. Just not used to see that anymore. If it's consistent, good for me.